Sources/Sentry/SentrySessionReplayIntegration.m

```diff
@@ -281,6 +281,10 @@ - (void)startWithOptions:(SentryReplayOptions *)replayOptions
         = (NSInteger)(shouldReplayFullSession ? replayOptions.sessionSegmentDuration + 1
                                               : replayOptions.errorReplayDuration + 1);
 
+    
+    dispatch_queue_attr_t attributes = dispatch_queue_attr_make_with_qos_class(                                                                               DISPATCH_QUEUE_SERIAL, DISPATCH_QUEUE_PRIORITY_LOW, 0);
+    SentryDispatchQueueWrapper* dispatchQueue = [[SentryDispatchQueueWrapper alloc] initWithName:"Sentry Session Replay" attributes:attributes];
```

Comment:
`m`: We should align the dispatch queue names with the Apple naming scheme for queues. I'm going to open a PR to rename the other queues.

![CleanShot 2024-10-15 at 13 47 01@2x](https://github.com/user-attachments/assets/76a13fda-bb64-4861-90f7-40019544ffba)


```suggestion
    SentryDispatchQueueWrapper* dispatchQueue = [[SentryDispatchQueueWrapper alloc] initWithName:"io.sentry.session-replay" attributes:attributes];
```

Reply:
PR to algin the dispatch queue names: https://github.com/getsentry/sentry-cocoa/pull/4442

Reply:
Makes sense